### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by ridabensalem**
### Portfolio mobile version

 in this version we :

- added the works section
- added about my self section
- added buttons actions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,19 +37,220 @@ <h2 class="supportive-text">
...
+                          </ul>
+                          <ul class="language-list">
+                                <ul class="js">
+                                  <li><img src="Images/Js.svg" alt="javascript-icon"></li>
+                                  <li><h5>javaScript</h5></li>
+                                </ul>
+                                <ul class="html">
+                                  <li><img src="Images/HTML.svg" alt="Html-icon"></li>
+                                  <li><h5>HTML</h5></li>
+                                </ul>
```
Comment (emmiiorji) [inline index.html:227]: Well done for rightly using the ul and li tags for representing the languages and skills. Kindly add some margin between the languages and frameworks to match the design. 

Also, kindly change the second `framework` to `skills`.

Your design:
image


Figma design:
image

File: index.html [hunk 2]
```diff
@@ -36,19 +37,220 @@ <h2 class="supportive-text">
...
+                        <div class="description-tags">
+                            <span class="first-tag">CANOPY</span>
+                            <img src="Images/Counter-1.png" alt="counter icon" class="counter">
+                            <span class="second-tag">Back End Dev</span>
+                            <img src="Images/Counter-1.png" alt="counter icon">
+                            <span class="third-tag">2015</span>
+                            </div>
+                            <div class="details">
+                                <p>
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
```
Comment (emmiiorji) [inline index.html:100]: Kindly reduce the margin between the cards a little bit in order to match the design template.

Your design:
image


Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @ridabensalem, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ridabensalem) [reply index.html:227]: I have applied everything you have suggested Thank you

File: index.html [hunk 3]
```diff
@@ -36,19 +37,220 @@ <h2 class="supportive-text">
...
+                          </div>
+                          <div class="button-action">
+                            <button>See Project</button>
+                          </div>
+                      </div>
+                    </div>
+                <!-- -----------------------------------------------Card number 2---------------------------------------- -->
+                                                                         
+                  
+                  <div class="card-works">
```
Comment (Nyame-Wolf) [inline index.html:87]: [OPTIONAL] I suggest you use the semantic tags `<article>`. A `div` tag is pretty general. This cards represent standalone parts of the application and they are articles of the section. It makes your code better and friendlier to screen readers

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @ridabensalem  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Linters are passing
- Great work using correct github flow
- Great communication with reviewer.

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**